autosklearn/automl.py

```diff
@@ -1492,7 +1508,9 @@ def fit_ensemble(
             backend=copy.deepcopy(self._backend),
             dataset_name=dataset_name if dataset_name else self._dataset_name,
             task=task if task else self._task,
-            metric=self._metric,
+            metric=(
```

Comment:
why not let the EnsembleBuilderManager decide which and how many metric to use?

Reply:
I would add that after merging #1434 to reduce the number of merge conflicts.